### ISSUE TITLE
Add valueOf(BigDecimal) and toBigDecimal() methods in Decimal.java

### ISSUE THREAD

**Issue by anthofo** [ta4j/ta4j#115]
I have the OHLC (ticks) data provided with BigDecimal values. To create a ta4j Tick I need to convert the values into Decimal.java with the valueOf(String): 

```
BigDecimal low; // ohlc data
...
// Create a Tick
Decimal.valueOf(low.toString())
...
```
There is a constructor that uses a BigDecimal but it is private.

And then if I want to use a BigDecimal from an indicator, I need to convert the Decimal to String, and then convert the String into BigDecimal:
```
return new BigDecimal(new SMAIndicator(new ClosePriceIndicator(series), 5).getValue(42).toString())
```

It can be boilerplate considering that `Decimal` is backed by a `BigDecimal`, but I don't know if it is on purpose or not.

**Comment by team172011** [comment]
You are right. It should be possible to convert `BigDecimal` to `Decimal` and the other way around. There is also a big discussion about the `BigDecimal` handling  #113 (#75, #88). Would be nice to know your provider. Are you belong to the users that really needs the high precision of  `BigDecimal`?

**Comment by anthofo** [comment]
Data are provided by https://github.com/timmolter/xchange which provides a common API.

**Comment by team172011** [comment]
fixed with #116

**Comment by team172011** [comment]
#116 merged
